### PR TITLE
TDP-4688 adding encoding to GET recordings

### PR DESCRIPTION
**Pull request by scapps-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

## Brief Summary of changes

clarifies the format and the need for encoding on the query params, considering the change to Slingshot and VMS where requests are now properly encoded from Slingshot to VMS

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4688.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

File: voice/methods/recordings/getRecordings.md [hunk 1]
```diff
@@ -15,10 +15,12 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 
 ### Supported Parameters
 
+All query parameters must be URL encoded.
```
Comment (n-miles) [inline voice/methods/recordings/getRecordings.md:18]: Maybe
```suggestion
**Note**: All query parameters must be URL encoded.
```
?

**Pull request merged**